### PR TITLE
Add ghost follow links to blob telepathy

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds ghost follow links to blobbernaut/overmind conversations. As an observer, you'll see this link to jump to the blob or naut who's speaking.

## Why It's Good For The Game
As a ghost, it's very nice to be able to follow who's talking in chat, and this is one of the locations that's notably missing a follow link.

## Images of changes
image

## Testing
- Spawn as blobbernaut
- Open the textbox
- type something
- ghost mid-typing
- send
- see message pop up with follow link
- tada

## Changelog
:cl:
tweak: Blob/bernaut telepathy now has a ghost follow link for observers.
/:cl:

### CODE REVIEW

**Comment by KittyNoodle** [comment]
would it also be possible for overminds to be able to click to the location of visible blobbers, often blobbers will ask for a location sensitive thing and its hard to locate where they are

File: code/game/gamemodes/blob/blobs/blob_mobs.dm [hunk 1]
```diff
@@ -224,8 +224,11 @@
 
 /mob/living/simple_animal/hostile/blob/blobbernaut/proc/blob_talk()
 	var/message = input(src, "Announce to the overmind", "Blob Telepathy")
-	var/rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([overmind])</span> <span class='message'>states, \"[message]\"</span></span></i></font>"
+	var/rendered
+	var/follow_text
 	if(message)
 		for(var/mob/M in GLOB.mob_list)
+			follow_text = isobserver(M) ? " ([ghost_follow_link(src, ghost=M)])" : ""
```
Comment (Sirryan2002) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm:231]:
```suggestion
			follow_text = isobserver(M) ? " ([ghost_follow_link(src, ghost = M)])" : ""
```

File: code/game/gamemodes/blob/blobs/blob_mobs.dm [hunk 2]
```diff
@@ -224,8 +224,11 @@
...
 /mob/living/simple_animal/hostile/blob/blobbernaut/proc/blob_talk()
 	var/message = input(src, "Announce to the overmind", "Blob Telepathy")
-	var/rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([overmind])</span> <span class='message'>states, \"[message]\"</span></span></i></font>"
+	var/rendered
+	var/follow_text
 	if(message)
 		for(var/mob/M in GLOB.mob_list)
+			follow_text = isobserver(M) ? " ([ghost_follow_link(src, ghost = M)])" : ""
+			rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([overmind])</span>[follow_text] <span class='message'>states, \"[message]\"</span></span></i></font>"
 			if(isovermind(M) || isobserver(M) || istype((M), /mob/living/simple_animal/hostile/blob/blobbernaut))
```
Comment (Sirryan2002) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm:233]:
```suggestion
			if(isovermind(M) || isobserver(M) || istype(M, /mob/living/simple_animal/hostile/blob/blobbernaut))
```

File: code/game/gamemodes/blob/overmind.dm [hunk 3]
```diff
@@ -94,12 +94,13 @@
...
 
-	var/verb = "states,"
-	var/rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([blob_reagent_datum.name])</span> <span class='message'>[verb] \"[message]\"</span></span></i></font>"
-
+	var/rendered
+	var/follow_text
 	for(var/mob/M in GLOB.mob_list)
+		follow_text = isobserver(M) ? " ([ghost_follow_link(src, ghost=M)])" : ""
+		rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([blob_reagent_datum.name])</span>[follow_text] <span class='message'>states, \"[message]\"</span></span></i></font>"
 		if(isovermind(M) || isobserver(M) || istype((M), /mob/living/simple_animal/hostile/blob/blobbernaut))
```
Comment (Sirryan2002) [inline code/game/gamemodes/blob/overmind.dm:102]:
```suggestion
		if(isovermind(M) || isobserver(M) || istype(M, /mob/living/simple_animal/hostile/blob/blobbernaut))
```

**Review by S34NW — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**